app/grants/clr.py

```diff
@@ -410,14 +355,18 @@ def populate_data_for_clr(grants, contributions, phantom_funding_profiles, clr_r
                 profile_contributions = contribs.filter(profile_for_clr__id=profile_id)
                 sum_of_each_profiles_contributions = float(sum([c.subscription.amount_per_period_usdt * clr_round.contribution_multiplier for c in profile_contributions if c.subscription.amount_per_period_usdt]))
                 phantom_funding = grant_phantom_funding_contributions.filter(profile_id=profile_id)
+
+                # get trust bonus
+                # # # MIGHT HAVE TO TURN THIS INTO A DISTINCT SET
+                profile_trust_bonus = profile_contributions.profile_for_clr.trust_bonus
```

Comment:
i think we could simply get first contribution in this list and get the profile's trust_bonus score 

https://docs.djangoproject.com/en/3.1/ref/models/querysets/#django.db.models.query.QuerySet.first